module/app/upgrades/v3/README.md

```diff
@@ -0,0 +1,17 @@
+# v3 upgrade
+
+This upgrade moves the gravity module from consensus version 2 to 3.
+
+## Summary of changes
```

Comment:
This release should also have updates to the SDK and IBC versions as compared to v2. Also, does this contain the address sorting bugfix for valsets?

Reply:
Yes it does; figured I missed some.